Sources/PackageDescription/PackageDescription.swift

```diff
@@ -309,13 +318,17 @@ public final class Package {
 
 /// A wrapper around an IETF language tag.
 ///
-/// To learn more about the IETF worldwide standard for language tags, see [RFC5646](https://tools.ietf.org/html/rfc5646).
+/// To learn more about the IETF worldwide standard for language tags, see
+/// [RFC5646](https://tools.ietf.org/html/rfc5646).
 public struct LanguageTag: Hashable {
 
     /// An IETF language tag.
     public let tag: String
 
     /// Creates a language tag from its IETF string representation.
+    ///
+    /// - Parameters:
+    ///   - tag: The string representation of an IETF language tag.
```

Comment:
I think it's better to use the compact "- Parameter <parameter>: <description>" form to document single-parameter functions.

Same for the few instances below in the same file.

Reply:
I went through, and I think I got all of them.